src/Commands.cpp

```diff
@@ -266,6 +264,10 @@ HRESULT DoStart(const std::wstring& sessionName, bool cpuSampling, VerbosityLeve
         options.SystemEventFlags |= TRACING_SESSION_SYSTEM_EVENT_FLAGS_CPU_SAMPLES;
     }
 
+    if (!admin) {
+        options.SystemEventFlags = 0;
+    }
+
     std::wcout << L"Starting tracing session " << sessionName << L"..." << std::endl;
 
     auto rc = StartTracingSession(sessionName.c_str(), options);
```

Comment:
`PrintError` should be modified to handle the new `RESULT_CODE_FAILURE_NO_CONTEXT_INFO_AVAILABLE` error code.

Reply:
I just added this